storage/scratchpad-benchmark/src/lib.rs

```diff
@@ -2,17 +2,35 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use anyhow::Result;
+use byteorder::{BigEndian, WriteBytesExt};
 use diem_config::config::RocksdbConfig;
-use diem_types::{account_address::HashAccountAddress, account_state_blob::AccountStateBlob};
+use diem_types::{
+    account_address::{AccountAddress, HashAccountAddress},
+    account_state_blob::AccountStateBlob,
+};
 use diemdb::DiemDB;
-use diemdb_benchmark::{gen_account_from_index, gen_random_blob};
 use executor_types::ProofReader;
 use rand::Rng;
 use std::{collections::HashMap, path::PathBuf};
 use storage_interface::DbReader;
 
 type SparseMerkleTree = scratchpad::SparseMerkleTree<AccountStateBlob>;
 
+fn gen_account_from_index(account_index: u64) -> AccountAddress {
```

Comment:
How about deleting "scratchpad-benchmark" entirely? :trollface: (It's covered by `storage/scratchpad/benches`.)

Reply:
Yeah, if it's covered then we can remove it.